latest/Dockerfile

```diff
@@ -44,7 +44,17 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     rm -rf /var/lib/apt/lists/*
 
 # Get required python packages
-RUN pip install pyserial dependency-injector
+RUN pip install pyserial dependency-injector && \ 
```

Comment:
are you sure you need this specific version for all these packages (and that you need to manually resolve these dependencies)?

Reply:
Most of those versions are the latest ones that still support Python 2, which is why we specified them

Reply:
I also think it may be preferable to install many of these via apt-get instead of pip. @davidAmiron?

Reply:
ah I see, wouldn't pip automatically do that?

Reply:
I’m not sure, when we left it unspecified, pip install would try to download the latest version and throw an error saying python 3 was required.

Reply:
Also I think the version numbers might help ensure that updates to dependencies don’t break the code, since it’s already pretty shaky given the whole python 2 vs. 3 thing